source/2019-07-19-the-ember-times-issue-107.md

```diff
@@ -33,69 +34,90 @@ responsive: true
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [Custom Configuration of Ember Auto Import for Your Project 🐹](https://github.com/ef4/ember-auto-import/tree/v1.3.0#customizing-build-behavior)
+
+The useful addon [Ember Auto Import](https://emberobserver.com/addons/ember-auto-import) helps you to use third-party JavaScript libraries in your Ember app easily by providing a **zero-config import** mechanism for **npm** modules.
+
+You can [configure the addon to fit your project's specific needs](https://github.com/ef4/ember-auto-import#customizing-build-behavior). By providing the `autoImport` configuration object in your app options defined in `ember-cli-build.js`, you can, f.e., change the name of the packages to import from or exclude packages to be imported with Ember Auto Import, which you might have already provided to your app in another way.
```

Comment:
I feel a bit of a contradiction between the `zero-config` and this sentence.

Reply:
Good point! I shortened the previous sentence to de-emphasise the zero config bit, so that the configuration section makes more sense just as you suggested :sparkles: